OWML.Common/IModInputHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace OWML.Common
+{
+	public interface IModInputHandler
+	{
+		int RegisterCombo(IModCombination combo);
+		int UnregisterCombo(IModCombination combo);
+		void _RegisterGamesBinding(InputCommand binding);
+		void _UnregisterGamesBinding(InputCommand binding);
+		bool _ShouldIgnore(KeyCode key);
```

Comment:
Why are some methods starting with _?

Reply:
removed from interface and changed to internal